README.md

```diff
@@ -34,14 +34,24 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. What is the difference between an adaptive website and a fully responsive website?
 
+The responsive website adjusts depending on what screen you are using, like tablet, desktop, phone, etc. The adaptive depends on what screen size is being used and is fixed, you have to write out each design individually.
+
 2. Describe what it means to be mobile first vs desktop first.
 
+mobile first starts by designing for the mobile phone, and scales larger to desktop. Desktop first is the exact opposite.
+
```

Comment:
mobile first uses (min-width ) for media queries.  desktop first uses (max-width) media queries